library/core/src/ptr/mut_ptr.rs

```diff
@@ -2361,6 +2367,80 @@ impl<T: ?Sized> PartialOrd for *mut T {
 #[unstable(feature = "kani", issue = "none")]
 mod verify {
     use crate::kani;
+    use core::mem;
+    // Constant for array size used in all tests, for performance reason
+    const ARRAY_SIZE: usize = 5;
```

Comment:
Isn't 5 too low? How is the verification performance numbers?

Reply:
Explained in another [PR's comment](https://github.com/model-checking/verify-rust-std/pull/166#discussion_r1861482793) that 5 is enough for this case.
This `performance` statement is outdated, and has been removed by newest commit.